locales/uk-UA.json

```diff
@@ -0,0 +1,2149 @@
+{
+  "posix_locale": "uk_UA.UTF-8",
+  "aliases": "ukua",
```

Comment:
aliases should be an array - the value should be "uk"

Reply:
Good catch.  Fixed.